initfiles/componentfiles/configschema/xsd/types.xsd

```diff
@@ -77,16 +78,21 @@
     </xs:simpleType>
 
     <xs:attributeGroup name="computerNodeReference">
-        <xs:attribute name="computer" hpcc:displayName="Computer" type="nodeName" use="required" hpcc:sourceKey="computer_name"
-            hpcc:mirrorFrom="/Environment/Hardware/Computer[@name]" hpcc:valueLimitRuleType="addDependencies_FromSiblingAttributeValue"
-            hpcc:valueLimitRuleData="/Environment/Hardware/Computer,name,netAddress,netAddress"/>
-        <xs:attribute name="netAddress" hpcc:displayName="Net Address" type="ipV4Address" use="required" hpcc:readOnly="true"
-            hpcc:sourceKey="computer_netaddress" hpcc:mirrorFrom="/Environment/Hardware/Computer[@netAddress]"/>
+        <xs:attribute name="computer" hpcc:displayName="Computer" type="nodeName" use="required"
+                      hpcc:sourceKey="computer_name"
```

Comment:
What is hpcc:sourceKey? Also please document it.

Reply:
Documented in the Wiki